packages/ketcher-core/src/application/editor/actions/template.ts

```diff
@@ -21,7 +21,7 @@ import { fromAtomsAttrs, mergeSgroups } from './atom';
 import { fromBondStereoUpdate, fromBondsAttrs, fromBondAddition } from './bond';
 
 import { Action } from './action';
-import closest from '../shared/closest';
```

Comment:
I just realized there is a problem – we can import `ketcher-core` from within `ketcher-react`, but not the other way around

So essentialy rather than having `closest.ts` in `ketcher-react` we should move it to `ketcher-core` and readjust all the import/export expressions related to that